marimbabot_msgs/actions/HitSequence.action

```diff
@@ -0,0 +1,15 @@
+#goal definition
+std_msgs/String[] notes_sequence # list of note string, e.g. c4
```

Comment:
We decided on sending a lilypond string, thus, everything that is considered in our data set (including tempo, slurs etc.) should be possible to extract from there. Considering the current feedback and result (played_sequence, number of played notes), as it is defined to date, what we have in https://github.com/UHHRobotics22-23/marimbabot/blob/main/marimbabot_planning/src/lilypond_to_hitpoint.cpp comes in handy because it already uses a a list for each hit/bar on the marimba. What are your opinions on that?